src/WebAssert.php

```diff
@@ -319,10 +319,9 @@ public function pageTextNotMatches($regex)
     public function responseContains($text)
     {
         $actual = $this->session->getPage()->getContent();
-        $regex = '/'.preg_quote($text, '/').'/ui';
         $message = sprintf('The string "%s" was not found anywhere in the HTML response of the current page.', $text);
 
-        $this->assert((bool) preg_match($regex, $actual), $message);
+        $this->assert(strpos($actual, $text) !== false, $message);
```

Comment:
The `i` modifier of `preg_match` was to match in case-insensitive manner, so to replicate that you need to use `stripos` instead of `strpos`.

Reply:
Yep, should've notice when I removed it.